gateway-ha/src/main/java/io/trino/gateway/ha/security/LbFilter.java

```diff
@@ -41,6 +44,7 @@ public void filter(final ContainerRequestContext requestContext) throws WebAppli
       }
 
     } catch (Exception e) {
+      log.error("Error while filtering request for authentication", e);
```

Comment:
I think this should probably be `log.debug` to avoid filling up the logs when someone supplies incorrect credentials. Alternatively, maybe a more specific exception can be thrown if `authenticate` returns false, so that bad credentials can be segregated from other exceptions.